src/physics/em/detail/SBPositronXsCorrector.i.hh

```diff
@@ -37,14 +37,24 @@ CELER_FUNCTION real_type SBPositronXsCorrector::operator()(Energy energy) const
 {
     CELER_EXPECT(energy > zero_quantity());
     CELER_EXPECT(energy.value() < inc_energy_);
-    real_type delta = cutoff_lorentz_
-                      - this->calc_lorentz_factor(energy.value());
+    real_type delta = cutoff_invbeta_
+                      - this->calc_invbeta(energy.value());
     return std::exp(alpha_z_ * delta);
 }
 
 //---------------------------------------------------------------------------//
 /*!
- * Calculate the lorentz factor (beta) of the positron for a gamma energy.
+ * Calculate the inverse of the relativistic positron speed.
+ *
+ * The input here is the exiting gamma energy, so the positron energy is the
+ * remainder from the incident energy. The relativistic speed \f$ \beta \f$
+ * is:
+ * \f[
+  \beta^{-1}(K) = \frac{K + m c^2}{\sqrt{K^2 + 2 K m c^2}}
+   = \frac{K + mc^2}{\sqrt{K^2 + 2 K mc^2 + (mc^2)^2 - (mc^2)^2}}
+   = \frac{K + mc^2}{\sqrt{(K + mc^2)^2 - mc^2}}
+   = 1/\sqrt{1 - \left( \frac{mc^2}{K + mc^2} \right)^2}
```

Comment:
This looks good now. Thanks for double checking it.
Now, is there any reason for reaching this final expression, given that this specific function is returning `1 / \beta = E / pc`?

Reply:
Looks like I can't suggest a change in an unchanged line, so I'll add the suggestion here:

Replace the return by:
```
return (energy + positron_mass_)
           / std::sqrt(ipow<2>(energy) + 2 * energy * positron_mass_);
```

Reply:
The final expression  is the one that was implemented in Geant4.  I was working from three different directions—the SB paper, the Geant4 implementation, and the definition of Beta from the ParticlelTrackView -- so that's why there's so much simple algebra there 😄

Reply:
Oh, ok, that makes sense now.

Reply:
And yep the return value can be improved -- changed it to be one `multiply` fewer.